Podfile

```diff
@@ -35,7 +35,7 @@ def aztec
     #pod 'WordPress-Editor-iOS', :git => 'https://github.com/wordpress-mobile/AztecEditor-iOS.git', :commit => ''
     #pod 'WordPress-Editor-iOS', :git => 'https://github.com/wordpress-mobile/AztecEditor-iOS.git', :tag => ''
     #pod 'WordPress-Editor-iOS', :path => '../AztecEditor-iOS'
-    pod 'WordPress-Editor-iOS', '1.19.7'
+    pod 'WordPress-Editor-iOS', '1.19.8'
```

Comment:
I had to bump the version of `WordPress-Editor-iOS` because it was complaining when installing the pods. This is most likely related to the changes introduced and afterward reverted in the following PRs:
- https://github.com/wordpress-mobile/WordPress-iOS/pull/17844
- https://github.com/wordpress-mobile/WordPress-iOS/pull/17989

cc @twstokes

Reply:
Hi @fluiddot 👋  - if there's something I can do to help just let me know.

Reply:
Hey @twstokes, I'm considering also adding an entry to the release notes as we had in https://github.com/wordpress-mobile/WordPress-iOS/pull/17844, since I had to bump the `WordPress-Editor-iOS` as part of this PR, wdyt?

Reply:
**To close the loop:** The release notes have been updated in https://github.com/wordpress-mobile/WordPress-iOS/pull/18065/commits/b87cd1366378a37e772925f692a19b5aed499f8b.